en/docs/learn/integration-tutorials/service-catalog-tutorial.md

```diff
@@ -432,39 +426,15 @@ You will now see the deployment as the first revision of the API:
 
 <a href="{{base_path}}/assets/img/integrate/tutorials/service-catalog/api-gateway-deployment-summary.png"><img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/api-gateway-deployment-summary.png" alt="api first revision"></a>
 
-### Step 8 - Publish the API
+### Step 7: Publish the API
 
 Go to the API overview in the **Publisher** portal and click **Publish** for the `HealthcareAPI` as shown below.
 
 <a href="{{base_path}}/assets/img/integrate/tutorials/service-catalog/api-overview-publish.png"><img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/api-overview-publish.png"></a>
 
 The API is now available in the **Developer** portal for consumers to access.
 
-### Step 9 - Subscribe to the API
-
-Now, let's assume you are an API consumer who wants to use the API. As a consumer, you need to first subscribe to the API.
-
-1.  Sign in to the **Developer** portal: `https://localhost:9443/devportal/apis`. 
-
-    !!! Tip
-        Use `admin` as the username and password.
-
-2.  Go to the **API** tab. The `HealthcareAPI` is listed as shown below.
-
-    <a href="{{base_path}}/assets/img/integrate/tutorials/service-catalog/developer-portal-api-list.png"><img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/developer-portal-api-list.png"></a>
-
-3.  Select the `HealthcareAPI` to open the API overview.
-
-    <a href="{{base_path}}/assets/img/integrate/tutorials/service-catalog/developer-portal-api-overview.png"><img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/developer-portal-api-overview.png"></a>
-
-4.  Go to the **Subscriptions** tab and subscribe using the **DefaultApplication** as shown below.
-
-    <a href="{{base_path}}/assets/img/integrate/tutorials/service-catalog/developer-portal-api-subscription.png"><img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/developer-portal-api-subscription.png"></a>
-
-!!! Tip
-    For detailed instructions, see [Subscribe to an API](https://apim.docs.wso2.com/en/4.2.0/consume/manage-subscription/subscribe-to-an-api).
-
-### Step 10 - Use the API
+### Step 8: Use the API
```

Comment:
Let's rephrase this line (444): `Open a terminal, navigate to the location where you saved the back-end service.`
as `Open a terminal and navigate to the location of the back-end service.`

Reply:
Fixed with [dcbc2d1](https://github.com/wso2/docs-mi/pull/984/commits/dcbc2d1cb3e0dd38f951300393aca407744f0acd)